wp-content/themes/core/components/blocks/gallery_grid/css/dialog.pcss

```diff
@@ -0,0 +1,268 @@
+        width: 50px;
+        opacity: 0.5;
+
+        &:focus,
+        &:hover {
+            background-color: var(--color-white);
+            opacity: 1;
+            transition: var(--transition);
+            
+
+            &:after {
+                color: var(--color-black);
+            }
+        }
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    .swiper-button-next,
+    .swiper-container-rtl .swiper-button-prev,
+    .swiper-button-prev,
+    .swiper-container-rtl .swiper-button-next {
+        background: transparent;
+        color: var(--color-white);
+        display: none;
+
+        &:before {
+            display: none;
+        }
+
+        &:after {
+            color: var(--color-white);
+            content: var(--icon-chevron-right);
+            font-family: var(--font-family-core-icons);
+            font-size: 22px;
+        }
+
+        @media (--viewport-full) {
+            display: flex;
+        }
+    }
+
+    .swiper-button-next,
+    .swiper-container-rtl .swiper-button-prev {
+        right: -102px;
+
+        &:after {
+            content: var(--icon-chevron-right);
+        }
+    }
+
+    .swiper-button-prev,
+    .swiper-container-rtl .swiper-button-next {
+        left: -102px;
+
+        &:after {
+            content: var(--icon-chevron-left);
+        }
+    }
+
+    .swiper-pagination-fraction {
+        font-size: 16px;
+    }
+}
+
+.c-dialog[aria-hidden="true"] {
+    display: none;
+}
+
+.c-dialog__overlay {
+    background-color: #151515;
+    bottom: 0;
+    color: var(--color-white);
+    height: 100%;
+    left: 0;
+    overflow: scroll;
+    position: fixed;
+    right: 0;
+    top: 0;
+    width: calc(100% + 20px);
+    z-index: 99998;
+
+    .admin-bar & {
+        top: var(--spacer-wpadmin-mobile);
+
+        @media (--viewport-wpadmin) {
+            top: var(--spacer-wpadmin-desktop);
+        }
+    }
+}
+
+.c-dialog__overlay-inner {
+    height: calc(100% - 68px);
+    padding-top: 100px;
+    position: absolute;
+    top: 68px;
+    width: 100%;
+
+    @media (--viewport-wpadmin) {
+        align-items: center;
+        display: flex;
+        padding-top: 0;
+    }
+}
+
+.c-dialog__header {
+    position: absolute;
+    top: 0;
+    left: 0;
+    padding: 20px;
+    width: 100%;
+    height: 68px;
+    background: #151515;
+    z-index: 10;
+    border-bottom: 1px solid var(--color-white);
+}
+
+.c-dialog__title {
+    font-size: 18px;
+    text-align: center;
+    text-transform: capitalize;
+}
+
+.c-dialog__content-wrapper {
+    position: relative;
+
+    @media (--viewport-medium) {
+        width: 100%;
+        height: 100%;
+        overflow: auto;
+    }
+}
+
+.c-dialog__content-inner {
+    margin: 0 auto;
+    padding: 0;
+    max-width: 1000px;
+    width: 100%;
+
+    @media (--viewport-full) {
+        width: 80%;
+        padding: 0 20px;
+    }
+}
+
+.c-dialog__close-button {
+    background-color: transparent;
```

Comment:
Imho, I think this should be abstracted to a unique "close" action style that we can use throughout the system rather than specific to the dialog here.